achyut/src/main/java/com/lftechnology/runner/Executer.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.runner;
+
+import java.util.Scanner;
+
+public class Executer {
+
+    private static Scanner userInput;
+    public static int chosenOption;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static chosenOption" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make chosenOption a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)